module/purpose/ScriptTuner/src/ScriptTuner.hpp

```diff
@@ -46,6 +46,17 @@ namespace module::purpose {
      */
     class ScriptTuner : public NUClear::Reactor {
     private:
+        /// @brief The autosave interval in seconds
+        /// @note Set to 0 to disable autosave
+        static constexpr size_t AUTOSAVE_INTERVAL = 30;
+
+        /// @brief Should we autosave to the local script directory, or to the
+        /// mounted ../NUbots folder in the docker container
+        bool autosave_on_robot;
```

Comment:
generally we initalise all the variables in the .hpp then we can be sure everything is initalised, and you don't have to use up lines in the cpp initialiser list

```suggestion
        bool autosave_on_robot = false;
```